cmd/fixer.go

```diff
@@ -497,6 +497,15 @@ var contentMismatch64Kfixer = &cobra.Command{
 		if len(args) == 0 {
 			return cmd.Usage()
 		}
+		if err := config.Setup(cfgFile); err != nil {
```

Comment:
Why do we need to call `config.Setup` here? The call in `cmd/root.go` is not enough?

Reply:
I based the swift initialization on the linked-app function: https://github.com/cozy/cozy-stack/commit/d33e0af59bc7f1ed71454f3f22a8601e49329357

You are right, a `PersistentPreRunE` on cmd/fixer would certainly be better